classes/Tools.php

```diff
@@ -3466,6 +3466,73 @@ public static function arrayReplaceRecursive($base, $replacements)
         }
         return $base;
     }
+
+    /**
+     * @deprecated Deprecated since 1.7.0
+     * Return path to a Product or a CMS category
+     *
+     * @param string $url_base Start URL
+     * @param int $id_category Start category
+     * @param string $path Current path
+     * @param string $highlight String to highlight (in XHTML/CSS)
+     * @param string $type Category type (products/cms)
+     */
+    public static function getPath($url_base, $id_category, $path = '', $highlight = '', $category_type = 'catalog', $home = false)
+    {
+        Tools::displayAsDeprecated();
```

Comment:
When I used this function to warn the developers, it has been requested to remove it because of the warnings displayed on all pages.
See https://github.com/PrestaShop/PrestaShop/commit/cdb09e2d9be870e92d853049f1a723855290dcba commited by @julienbourdeau.

Because you warn explicitely in the PHP doc, we can remove it here.